data/remnant/remnant ships.txt

```diff
@@ -1183,33 +1188,30 @@ ship "Peregrine"
 		licenses
 			"Remnant Capital"
 		"cost" 41627000
-		"shields" 16400
-		"hull" 10600
+		"shields" 15400
+		"hull" 8600
 		"required crew" 9
 		"bunks" 25
 		"mass" 350
 		"drag" 4.3
 		"heat dissipation" 0.85
-		"fuel capacity" 1100
+		"fuel capacity" 900
 		"ramscoop" 3.25
-		"cargo space" 59
+		"cargo space" 49
 		"outfit space" 552
```

Comment:
This ought to be reduced at least a little, I think. The Peregrine just isn't making the outfit space tradeoff that other Remnant ships make for its suite of integrated features, nor is it making any sacrifices for its extreme speed. As an initial suggestion, I would suggest dropping it to 522. That, coupled with the other changes here, would bring its effective outfit space total down to 707, lower than the Model 128's 713 and hence actually in line with being a medium warship.